tests/compat-bpf.sh

```diff
@@ -96,28 +73,14 @@ begin_test "create table, chain and bpf rule"
 )
 end_test
 
-begin_test "validate nft output: xt field"
-(
-    $NFT_LIST_TABLE ip $TABLE
-    $NFT_LIST_TABLE_JSON ip $TABLE | $JQ .nftables[].rule.expr | grep xt
-)
-end_test
-
-begin_test "validate nft output: drop verdict"
+begin_test "python compare"
```

Comment:
We called this "json diff" in the other test, lets do that here and elsewhere too.